src/main/java/nextstep/fp/Conditional.java

```diff
@@ -0,0 +1,5 @@
+package nextstep.fp;
+
+public interface Conditional {
```

Comment:
```suggestion
@FunctionalInterface
public interface Conditional {
```
테스트에 functional interface로 활용하고 있다면 명시해주면 좋을 것 같아요 😄